README.md

```diff
@@ -7,20 +7,70 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet"
+          href="https://cdn.jsdelivr.net/npm/simple-icons-font/font/simple-icons.min.css"
+          type="text/css">
+  </head>
+  <body>
+    <i class="simpleicons simpleicons-simpleicons simpleicons--color"></i>
+  </body>
+</html>
+```
 
-### General Usage
+### Manual download
+
+1. Download the latest version of the package from
+[here][npm-registry-tarball-link], extract it and move the content of the
+`font` folder to your desired location. 
+1. Reference the CSS file in a `link` tag of your HTML:
+
+```html
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="simple-icons.min.css" type="text/css">
```

Comment:
i would only leave this part of the sample code in this section, and add that you may need to configure the path (and that you likely want it to be "absolute"):

```suggestion
    <link rel="stylesheet" href="/path/to/simple-icons.min.css" type="text/css">
```